writer.go

```diff
@@ -3,64 +3,189 @@ package log
 import (
 	"io"
 	"sync"
-	"time"
 )
 
+var MaxWriterBuffer = 16 * 1024 * 1024
```

Comment:
yeah, the whole point of this is to avoid backpressure entirely. If theres backpressure, then one of two things happens, we consume large amounts of memory, or the daemon locks up. I dont like either of those options, so dropping laggy connections is our only other option.